include/log.h

```diff
@@ -67,6 +67,8 @@ struct ofono_debug_desc {
 					__FILE__, __FUNCTION__ , ## arg); \
 } while (0)
 
+#define PRINTABLE_STR(s) ((s) ? (s) : "(null)")
```

Comment:
PRINT_STR suggests that it prints something, although I agree we could come up with a shorter name.